R-packages/covidcast/R/plot.R

```diff
@@ -252,6 +253,80 @@ plot_choro = function(x, time_value = NULL, include = c(), range,
   geo = df$geo
   names(val) = geo
 
+  # Make background layer for MSA and HRR maps which are incomplete
+  if ((attributes(x)$metadata$geo_type == "msa") ||
+      (attributes(x)$metadata$geo_type == "hrr")) {
+    map_df = sf::st_read(system.file(
+                         "shapefiles/state",
+                         "cb_2019_us_state_5m.shp",
+                         package = "covidcast"))
+    background_crs = sf::st_crs(map_df)
+    map_df$STATEFP <- as.character(map_df$STATEFP)
+    map_df$is_alaska = map_df$STATEFP == '02'
```

Comment:
I think it is possible and cleaner to wrap many of these column creations into one call to `dplyr::mutate()`.